wlauto/workloads/reader/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -112,7 +112,9 @@ private Timer selectLocalFilesList() throws Exception {
     private Timer selectSearchFileButton() throws Exception {
         // Click the button to search from the present file list view
         UiObject searchButton = getUiObjectByResourceId("com.adobe.reader:id/split_pane_search",
-                                                        "android.widget.TextView");
+                                                        "android.widget.TextView",
+                                                        (long) 10000 );
```

Comment:
Might be nice to have something like TimeUnit.SECONDS.toMillis(10) or a variable to reduce magic numbers. Minor point though.